docs/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Generated documentation
+book/
+doc/
```

Comment:
Since the entire `docs/` directory is included in the git commit that deploys the documentation this ignore file would cause the `book/` and `doc/` directories to be ignored. Only the `index.html` and `README.md` files would be included.

Reply:
Oh, interesting! I didn't realize the gitignore was used by GitHub pages to exclude certain content from being published. Thanks for pointing that out!

Reply:
GitHub Pages just published whatever is in the branch at the directory specified. But the deployment action uses git which will not commit ignored file. There is probably a workaround but I think the best solution is just to move these directories to the `.gitignore` file on the root directory.

Reply:
Yes, agreed. I'm making that change today. Thanks so much for clarifying!